corehq/apps/users/static/users/js/roles.js

```diff
@@ -359,8 +359,22 @@ hqDefine('users/js/roles',[
                 };
             },
         };
-        self.submitNewRole = function () {
-            // moved saveOptions inline
+        self.roleError = ko.observable("");
+        self.setRoleError = function (form, error) {
+            self.roleError(error);
+            $(form).find('.modal-footer').addClass('has-error');
```

Comment:
Why not use knockout bindings on the modal instead of introducing jQuery here?

Reply:
902bb6714f9